openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMUtils.java

```diff
@@ -57,9 +57,11 @@ static public void loadLibs() {
 
         if (!libsLoaded) {
             try {
-                loadSharedLibraryFromJar("libgomp.so.1.0.0");
-                loadSharedLibraryFromJar("lib_lightgbm.so");
-                loadSharedLibraryFromJar("lib_lightgbm_swig.so");
+                final String cpuArchName = System.getProperty("os.arch");
+                final CpuArchitecture cpuArchitecture = CpuArchitecture.valueOf(cpuArchName.toUpperCase());
```

Comment:
If a non-supported arch is used an exception will be thrown, however, I would catch it and add some meaningful message to it to help the users and then re-throw it.

Reply:
done